README.md

```diff
@@ -81,19 +81,17 @@ Planned features:
 
 ## 🌏 Browsers Support
 
-**We do our best to support recent versions of major browsers**
```

Comment:
If we want to keep such sentence, it will be better to 
- put it below the table
- not using bold (there is no reason to highlight such sentence so much).
- improve wording and provide more specific info (latest 2 versions ?)

Reply:
It's ok for me 👍🏻